server/src/main/kotlin/suwayomi/tachidesk/opds/impl/OpdsEntryBuilder.kt

```diff
@@ -348,8 +356,9 @@ object OpdsEntryBuilder {
                 OpdsLinkXml(
                     OpdsConstants.LINK_REL_ACQUISITION_OPEN_ACCESS,
                     "/api/v1/chapter/${chapter.id}/download?markAsRead=${serverConfig.opdsMarkAsReadOnDownload.value}",
-                    OpdsConstants.TYPE_CBZ,
+                    serverConfig.opdsCbzMimetype.value.mediaType,
                     MR.strings.opds_linktitle_download_cbz.localized(locale),
+                    length = cbzFileSize!!,
```

Comment:
We probably don't need to enforce non-null here.
```suggestion
                    length = cbzFileSize,
```